packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -3303,11 +3303,20 @@ describe('Checkout', () => {
                         }
                     });
 
+                    $cookies.set = jest.fn((name, value, expire) => {
+                        expect(name).toEqual('je-gtm-event');
+                        expect(value).toEqual('engagement|form_checkout_guest|success');
+                        const now = new Date();
+                        const expectedExpireDate = now.setMinutes(now.getMinutes() + 1);
+                        expect(expire).toBeCloseTo(expectedExpireDate, -1);
+                    });
+
                     // Act
                     await wrapper.vm.onFormSubmit();
 
                     // Assert
                     expect(updateMessageSpy).toHaveBeenCalled();
+                    expect($cookies.set).toHaveBeenCalled();
```

Comment:
Could you change the test description name to match the changes please? Or just add a new test.
And then can you do `toHaveBeenCalledWith` please?